.golangci.yml

```diff
@@ -14,6 +14,7 @@ linters:
   - misspell
   - nakedret
   - typecheck
+  - unconvert
```

Comment:
should we be adding comments to each of these saying what they do? for eslint you can just go to the eslint site and look up good documentation for a rule by name, but these are harder to find. for example, if I google "go lint unconvert" a direct link to https://github.com/mdempsky/unconvert doesn't even show up in the first page of results.

Reply:
good idea! i've updated `.golangci.yml` with some helpful pointers.